sklearn/linear_model/coordinate_descent.py

```diff
@@ -525,21 +525,21 @@ class ElasticNet(LinearModel, RegressorMixin, MultiOutputMixin):
 
     Parameters
     ----------
-    alpha : float, optional
+    alpha : float, optional, default 1.0
```

Comment:
I think we prefer the `default=1.0` format. Please refer to https://github.com/scikit-learn/scikit-learn/issues/14404 for the current discussion. We probably would move away from using `optional` too.

Reply:
@adrinjalali Thanks for the feedback. Yes, the format in #14404 is cleaner. I will make those changes.